client/app/lib/components/contentModal/styl/contentModal.styl

```diff
@@ -356,3 +361,8 @@ main.main-container
   button.minimize-button
     abs 97px 60px 0 0
     z-index 10
+    background-color      #fff !important
+    border                2px solid #CBCBCB
+    span
+      color               #858585 !important
+      line-height         25px
```

Comment:
@cansirin please create a common class name and apply to those both buttons. Also, remove unnecessary class names from the buttons so that you don't need `!important`s here as well.